### PR TITLE
Simplify for loop and string building

### PR DESCRIPTION
**Pull request by lucc**

### CODE REVIEW

**Comment by pazz** [comment]
this looks good to me. Shall we wait until after the release to merge this?

**Comment by lucc** [comment]
Ok with me.

File: alot/buffers.py [hunk 1]
```diff
@@ -167,15 +167,11 @@ def rebuild(self):
...
+            key_ids = []
             for key in encrypt_keys:
-                if key is not None:
-                    if first_key:
-                        first_key = False
-                    else:
-                        description += ', '
-                    if len(key.subkeys) > 0:
-                        description += key.uids[0].uid
+                if key is not None and len(key.subkeys) > 0:
```
Comment (dcbaker) [inline alot/buffers.py]: can you just use `if key is not None and key.subkeys`?

Comment (lucc) [inline alot/buffers.py]: sorry, obviously.

Comment (dcbaker) [inline alot/buffers.py]: no worries, thanks for changing it!

**Comment by dcbaker** [comment]
I went ahead and approved this since it looks good. But lets do like @pazz suggested and not merge it until after the release.

**Pull request merged**